.grabl/automation.yml

```diff
@@ -73,7 +73,7 @@ build:
         export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
         export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
         bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
-        bazel test //tests/behaviour/concept/... --test_output=errors --jobs=1
```

Comment:
Now that we randomise the port number, we can safely re-enable parallelisation of all tests (except the Connection tests, which intermittently cause Grabl to run out of memory if parallelised)